src/Apps/Collect/CollectApp.tsx

```diff
@@ -20,8 +19,7 @@ export interface CollectAppProps {
 export class CollectApp extends Component<CollectAppProps> {
   render() {
     return (
-      <>
-        <Title>Collect | Artsy</Title>
+      <div>
```

Comment:
I believe @l2succes intended for this to be a fragment instead of a `div`. Not sure if it even matters, but I wanted to point it out.

Reply:
I was assuming the fragment was so we could add meta tags -- now that we have a meta template we can use a div with no problem. Is there any reason we shouldn't wrap a top level app in a div?  I find the `<>` a confusing/non-idomatic syntax.

Reply:
@eessex Fragments are actually the way forward in this particular use case. Prior to React 16.2, we've used divs to wrap a set of components because it was the only way to return multiple elements. https://reactjs.org/docs/fragments.html

Also our top level apps are injected inside our [`Boot`](https://github.com/artsy/reaction/blob/master/src/Artsy/Router/Components/Boot.tsx#L60) component which already includes a container grid. If you look at the other apps, they all use a top level fragment for that reason as well

Reply:
Ok sounds good, what is the reason for using the syntax `<>` over `<Fragment>`?

Reply:
It’s the shorthand syntax. Perhaps we should you the regular syntax until people become more familiar with the pattern...? Because I had the same question.

Reply:
Yeah, the syntax is useful because they can provide logical JSX groupings without filling the HTML/RN DOM with filler elements